### PR TITLE
Use Dekorate yaml in e2e tests

### PR DESCRIPTION
**Pull request by gytis**
https://github.com/snowdrop/component-operator/issues/86

### CODE REVIEW

**Comment by gytis** [comment]
Is there a reason why only master branch was enabled for CircleCI? If so let me know and I'll change it back. But for now I removed the filter so that PRs could be tested

File: demo/scripts/k8s_push_start.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 set -x
 
+dir=$(dirname "$0")
```
Comment (geoand) [inline demo/scripts/k8s_push_start.sh]: Doesn't really matter, but maybe it should be `DIR` to be consistent with the rest?

**Comment by geoand** [comment]
It's hard to tell since I don't have all the context, but it seems like a reasonable change

File: demo/scripts/k8s_push_start.sh (hunk 1 above)
Comment (gytis) [reply demo/scripts/k8s_push_start.sh]: I made it consistent with the rest of the variables in this file. They're all lowercase for some reason

**Comment by gytis** [comment]
@geoand there are three changes:
1. To use component yaml files generated by Dekorate instead of hardcoded values.
2. Refactor scripts to use $(dirname "$0") instead of a hardcoded path to allow script execution from any location.
3. Enable CI jobs on any branches.

File: demo/scripts/k8s_push_start.sh (hunk 1 above)
Comment (geoand) [reply demo/scripts/k8s_push_start.sh]: OK

**Pull request merged**